test/check-application

```diff
@@ -1257,6 +1258,12 @@ class TestApplication(testlib.MachineCase):
         b.click('.pf-c-modal-box button:contains(Restore)')
         b.wait(lambda: self.getContainerAttr("swamped-crate", "State") in 'Running')
 
+        # A new MAC address should have been generated
+        version = self.execute(False, "podman -v").strip().split(' ')[-1]
+        # Fixed in podman 4.4.0 https://github.com/containers/podman/issues/16666
+        if int(version.split('.')[0]) >= 4 and int(version.split('.')[1]) > 3:
```

Comment:
This would fail for e. g. podman 5.0 or 5.2. I agree that right now a hardcoded `m.image` list is still too early, as 4.4 will most probably enter Fedora, Debian testing, arch etc. soon. I recommend to convert `version` to a tuple of ints and compare it as tuple, which works as intended. See e.g. `test/verify/check-storage-resize`:

    if self.storaged_version < [2, 7, 6]